pde2e-image/common/unix/common.sh

```
@@ -122,17 +122,17 @@ load_secrets() {
 
 # Clone repository and checkout specific branch
 clone_checkout() {
-    # Checkout Podman Desktop if it does not exist
     local_repo=$1
     local_fork=$2
     local_branch=$3
+    local_git_provider_url="${4:-https://github.com}"
     echo "Working Dir: $workingDir"
```

Shellcheck (0.11.0)

[warning] workingDir is referenced but not assigned.

(SC2154)